apps/website/src/pages/docs/tabs.mdx

```diff
@@ -9,27 +9,134 @@ group: core
 
 import PropsTable from '~/components/PropsTable.astro';
 import LiveExample from '~/components/LiveExample.astro';
-import Placeholder from '~/components/Placeholder.astro';
 import * as AllExamples from 'examples';
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='core-tabs' />
-
 <LiveExample src='Tabs.main.tsx'>
   <AllExamples.TabsMainExample client:load />
 </LiveExample>
 
 ## Usage
 
+### Composition API
+
+You can build Tabs with subcomponents that are fully customizable.
+
+```js
+<Tabs.Wrapper>
+  <Tabs.TabList>
+    <Tabs.Tab value='apple' label='Apple' key='apple' />
+    <Tabs.Tab value='orange' label='Orange' key='orange' />
+    <Tabs.Tab value='pear' label='Pear' key='pear' />
+  </Tabs.TabList>
+  <Tabs.Panel value='apple' key='apple'>
+    {/*...Content for apple*/}
+  </Tabs.Panel>
+  <Tabs.Panel value='orange' key='orange'>
+    {/*...Content for orange*/}
+  </Tabs.Panel>
+  <Tabs.Panel value='pear' key='pear'>
+    {/*...Content for pear*/}
+  </Tabs.Panel>
```

Comment:
nit: Would be easier to understand the DOM hierarchy with new lines.

```suggestion
  <Tabs.TabList>
    <Tabs.Tab value='apple' label='Apple' key='apple' />
    <Tabs.Tab value='orange' label='Orange' key='orange' />
    <Tabs.Tab value='pear' label='Pear' key='pear' />
  </Tabs.TabList>
  
  <Tabs.Panel value='apple' key='apple'>
    {/*...Content for apple*/}
  </Tabs.Panel>
  <Tabs.Panel value='orange' key='orange'>
    {/*...Content for orange*/}
  </Tabs.Panel>
  <Tabs.Panel value='pear' key='pear'>
    {/*...Content for pear*/}
  </Tabs.Panel>
```

Same for the other code markdown below.

Reply:
Updated